### ISSUE TITLE
Update price of sign without recreating shop

### ISSUE THREAD

**Issue by Shybella** [TheoKah/CarrotShop#52]
Implement a command or feature that allows you to change the price on the sign without having to recreate or link the sign to the chest to re update it with the new price.

**Comment by TheoKah** [comment]
I'll think of something.

I will only allow price change though. Not the type

**Comment by Shybella** [comment]
Yes, only price. Too much involved to change the type the player should do that themselves anyhow. 

People can change the price at anytime but doesn't update.

This would be a very handy feature.
